src/faq/general.rst

```diff
@@ -0,0 +1,55 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _faq/general:
+
+=======
+General
+=======
+
+What is CouchDB?
+----------------
+
+CouchDB is a document-oriented, NoSQL_ database. The
+:ref:`Introduction <intro>` provides a high level overview of the
+CouchDB system.
+
+.. _NoSQL: https://en.wikipedia.org/wiki/NoSQL
+
+What does "Couch" mean?
+-----------------------
+
+"Couch" is an acronym which stands for **C**\ luster **O**\ f **U**\ nreliable
+**C**\ ommodity **H**\ ardware. This is a statement of Couch's long-term goal of
+massive scalability and high reliability on fault-prone hardware. The
+distributed nature and flat address space of the database will enable node
```

Comment:
"enables" not "will enable," this is a vagary of the 1.x approach showing up in this FAQ